### DISCUSSION TITLE
新人入坑

### DISCUSSION THREAD

**Discussion by starrailandgenshinimpact** [share121/inter-knot#1176] [其他]
![1](https://github.com/user-attachments/assets/32d31520-8424-4c82-b46b-6612239e8ac8)
嘎嘎，机械，嘎嘎

**Comment by Jack1Nero1** [comment]
谁家应急食品布

**Comment by 10kaihua** [comment]
噢~！小家伙真漂亮，（就是为什么感觉不太聪明）
